tests/dummy/app/controllers/index.ts

```diff
@@ -65,6 +64,7 @@ import { chromeHacksPlugin } from '@lblod/ember-rdfa-editor/plugins/chrome-hacks
 import type { PluginConfig } from '@lblod/ember-rdfa-editor';
 import { emberApplication } from '@lblod/ember-rdfa-editor/plugins/ember-application';
 import { getOwner } from '@ember/application';
+import { heading } from '@lblod/ember-rdfa-editor/plugins/heading/nodes/heading';
```

Comment:
It feels strange to be using deprecated APIs in our own test app, should we use the `WithConfig` versions?

Reply:
Yeah that's fair, will update.